core-bundle/assets/controllers/tinymce-controller.js

```diff
@@ -1,15 +1,59 @@
 import { Controller } from '@hotwired/stimulus';
 
 export default class extends Controller {
-    static values = {
-        config: Object
+    initialize() {
+        Turbo.cache.exemptPageFromCache();
     }
 
     connect() {
-        tinymce?.init(this.configValue);
+        if(!this.element.tinymceConfig) {
+            console.error('No tinymce config was attached to the DOM element, expected an expando property called "tinymceConfig".', this.element);
+
+            return;
+        }
+
+        // Move the config to a member variable to prevent memory leaks.
+        this.config = {
+            ...this.element.tinymceConfig,
+            ...{
+                target: this.element
+            }
+        };
+
+        delete this.element.tinymceConfig;
+
```

Comment:
We could probably omit this part (now that I know how snapshots work), because it will never leak and then use `element.tinymceConfig` in L33 instead of `this.config`.